PSKoans/Koans/Introduction/AboutNumbers.Koans.ps1

```diff
@@ -61,6 +61,15 @@ Describe 'Basic Number Types' {
 Describe "Banker's Rounding" {
 
     It 'rounds to nearest even number on .5' {
+        <#
+            Rounding
+
```

Comment:
These lines are probably not needed here; I usually only include this for the topic name of the file itself at the top of a file. 🙂

Reply:
I've removed them 😄